### PR TITLE
Comment

### PR DESCRIPTION
**Pull request by pantageepapa**
Screenshot 2023-12-10 at 17 28 35

- Comment frontend with backend connected 
- It fetches comments when the comments are shown or a new comment is submitted 
- Submitting a comment with Enter Key is possible 

-> Sometimes the new comment is not shown directly after being submitted as the backend needs some time to update the database. I dont know how to fix this issue so if you have any ideas please let me know!

### CODE REVIEW

File: src/components/MainPage/Post/Post.jsx [hunk 1]
```diff
@@ -67,18 +171,51 @@ function Post({
...
+									type="text"
+									placeholder="What do you think?"
+									className="comment-input"
+									value={comment}
+									onKeyDown={handleKeyDown}
+									onChange={(e) => setComment(e.target.value)}
+								/>
+								<button
+									className="submit-comment"
+									onClick={handleCommentSubmit}>
```
Comment (Pullisepop) [inline src/components/MainPage/Post/Post.jsx:195]: handleCommentSubmit should have logic to not allow empty string to be passed when commenting. i tried entering empty comment and it crashes users session. i need to log out and in to proceed doing anything

File: src/components/ListCard/listcards.js [hunk 2]
```diff
@@ -1,52 +1,66 @@
...
+				if (personalLists[index].length > 5) {
+					truncatedText[5] = truncatedText.slice(5) + " and more";
+				}
+
+				return (
+					<Card
+						key={index}
+						title={title}
+						text={truncatedText.join(", ")}
+						imageSource={images[index]}
```
Comment (Pullisepop) [inline src/components/ListCard/listcards.js:29]: does not seem to return image for favorite restaurant. i chose retaurant that has an image in feeed, added to favs, and when checked favs, the image is not rendered.

File: src/pages/ProfilePage/ProfilePage.jsx [hunk 3]
```diff
@@ -5,7 +5,6 @@ import ProfilePicture2 from "../../components/ProfilePicture2/ProfilePicture2";
 import ButtonSh from "../../components/shared/ButtonShort/ButtonShort";
 import TopBar from "../../components/shared/TopBar/TopBar";
 import "./ProfilePage.css";
-import BackButton from "../../components/shared/GoBackButton/GoBackButton";
```
Comment (Pullisepop) [inline src/pages/ProfilePage/ProfilePage.jsx:8]: Is it neccessary to delete the BackButton? Are we sure we don't want to use it?

**Review by Pullisepop — CHANGES_REQUESTED** [review]
empty comment submit should be changed

File: src/components/MainPage/Post/Post.jsx (hunk 1 above)
Comment (pantageepapa) [reply src/components/MainPage/Post/Post.jsx:195]: Good point!

File: src/pages/ProfilePage/ProfilePage.jsx (hunk 3 above)
Comment (pantageepapa) [reply src/pages/ProfilePage/ProfilePage.jsx:8]: I deleted unused imports

**Review by lydiastenflo — APPROVED** [review]
Looks great! Excited to start commenting!

**Pull request merged**